spm12/metadata/get_metadata_val_classic.m

```diff
@@ -775,12 +777,13 @@
                     % adFree: [RefocCorr ScaleSGrad? MaxRefocAngle DecRefocAngle FAforReferScans]
                     parLocation{cRes} = [nam{1} '.adFree(3:4)'];
                     parValue{cRes} = val{1}.adFree(3):-val{1}.adFree(4):0;
-                case 'seste1d3d2' % 1mm protocol from WTCN (MFC)
+                case {'seste1h3d2' 'seste1f3d2' 'seste1e3d2'} % 1mm protocol from WTCN
                     % wip parameters are sorted as follows:
-                    % alFree: [VoxDeph,SpoilAmp,EddCurr0,EddCurr1,TRamp,TFlat,BWT,0,0,0,0,0,2,MixingTime,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,12345],
-                    % adFree: [0,0,0,0,0,0,0,SlabGradScale,RefocCorr,0,0,RFSpoilBasicIncr]
+                    % alFree: [VoxDeph,SpoilAmp,EddCurr0,EddCurr1,TRamp,TFlat,BWT,0,0,0,0,0,0,MixingTime,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,12345]
+                    % adFree: [0,0,0,0,0,0,0,SlabGradScale,RefocCorr,0,FatSatAngle,RFSpoilBasicIncr] --> 'seste1h3d2' 'seste1f3d2'
+                    % adFree: [0,0,0,0,0,0,0,SlabGradScale,RefocCorr,0,0,RFSpoilBasicIncr] --> 'seste1e3d2', i.e. no FatSatAngle in special card
                     parLocation{cRes} = 'HardCodedParameter';
-                    parValue{cRes} = 230:-10:0;
+                    parValue{cRes} = 2*(115:-5:65);
```

Comment:
Thanks for catching that.  You are right.  It should be 0 so that the correct number of values can be selected depending on how many measurements (SE/STE pairs) are available.  I assume that is what is done elsewhere in the code.  I will update the PR.

Reply:
Yes, it is dictated by the number of measurements.

Reply:
Thanks for the clarification.